requirements/caffe_tensorflow_keras_install.sh

```diff
@@ -1,12 +1,12 @@
 # Generic dependencies
 echo "Installing generic dependencies"
-apt-get install git libatlas-base-dev python-protobuf python-numpy python-scipy python-h5py unzip make libblas-dev liblapack-dev libatlas-base-dev gfortran python-pip python-dev
+sudo apt-get install git libatlas-base-dev python-protobuf python-numpy python-scipy python-h5py unzip make libblas-dev liblapack-dev libatlas-base-dev gfortran python-pip python-dev
```

Comment:
Might be good to add a `-y` flag here if the sudo is not used before. Saves the user's effort to manually approve the install.